core/templates/pages/topic-editor-page/services/entity-creation.service.spec.ts

```diff
@@ -77,9 +75,4 @@ describe('Entity creation service', function() {
 
     expect(spy).toHaveBeenCalled();
   });
-
-  it('should return subtopic Id from URL', function() {
-    $location.path('/subtopic_editor/2');
-    expect(TopicEditorRoutingService.getSubtopicIdFromUrl()).toEqual(2);
-  });
```

Comment:
functionality of `topic-editing-routing.service` should be tested in the serivce's spec file only.